mesonbuild/backend/xcodebackend.py

```diff
@@ -722,6 +722,7 @@ def generate_xc_build_configuration(self):
                 if links_dylib:
                     dep_libs = ['-Wl,-search_paths_first', '-Wl,-headerpad_max_install_names'] + dep_libs
                 dylib_version = None
+                cargs = []
```

Comment:
I know it's a nit, but can you put this closer to where it's used?

Reply:
updated